src/templates/community-champion-profile.js

```diff
@@ -257,6 +273,230 @@ const Bio = styled(P)`
     }
 `;
 
+const LinksTitle = styled(H6)`
+    color: ${({ theme }) => theme.colorMap.greyLightThree};
+    font-weight: bold;
+    letter-spacing: 0.1em;
+    margin-bottom: ${size.xsmall};
+    text-transform: uppercase;
+    @media ${screenSize.upToMedium} {
+        font-size: ${fontSize.small};
+        line-height: ${lineHeight.small};
+    }
+`;
+
+const BlogsAndPublicationsLinksContainer = styled('div')`
+    display: flex;
+    flex-direction: column;
+    row-gap: ${LINK_ROW_GAP};
+    width: ${LINK_CONTAINER_WIDTH};
+    @media ${screenSize.upToSmallDesktop} {
+        width: 100%;
+    }
+`;
+
+const BlogsAndPublicationsLink = styled(Link)`
+    font-size: ${fontSize.small};
+    line-height: ${lineHeight.small};
+    max-width: 100%;
+    width: max-content;
+    @media ${screenSize.upToSmallDesktop} {
+        width: 100%;
+    }
+`;
+
+const BlogsAndPublications = ({ blogsAndPublications }) => (
+    <div>
+        <LinksTitle collapse>Blogs &amp; Publications</LinksTitle>
+        <BlogsAndPublicationsLinksContainer>
+            {blogsAndPublications.map(({ id, title, link }) => (
+                <BlogsAndPublicationsLink key={id} href={link}>
```

Comment:
Could the `title` be the key? We then don't need to pull in ID at all

Reply:
i think ids are a better option here since they are guaranteed to be unique whereas titles aren't

Reply:
Swag